core/templates/dev/head/profile/profile.html

```diff
@@ -31,104 +31,91 @@
-            <md-content class="oppia-profile-tabs-content">
-              <div class="oppia-profile-explorations-container">
-                <exploration-summary-tile ng-repeat="expl in userCreatedExplorations"
-                    exploration-id="expl.id"
-                    exploration-title="expl.title"
-                    last-updated-msec="expl.last_updated_msec"
-                    objective="expl.objective"
-                    category="expl.category"
-                    ratings="expl.ratings"
-                    thumbnail-icon-url="expl.thumbnail_icon_url"
-                    thumbnail-bg-color="expl.thumbnail_bg_color"
-                    num-views="expl.num_views"
-                    is-community-owned="expl.community_owned">
-                </exploration-summary-tile>
-              </div>
-              <div ng-if="userCreatedExplorations.length === 0">
-                <em>This user hasn't created any explorations yet.</em>
-              </div>
-            </md-content>
-          </md-tab>
-          <md-tab label="Edited Explorations">
-            <md-content class="oppia-profile-tabs-content">
-              <div class="oppia-profile-explorations-container">
-                <exploration-summary-tile ng-repeat="expl in userEditedExplorations"
-                    exploration-id="expl.id"
-                    exploration-title="expl.title"
-                    last-updated-msec="expl.last_updated_msec"
-                    objective="expl.objective"
-                    category="expl.category"
-                    ratings="expl.ratings"
-                    thumbnail-icon-url="expl.thumbnail_icon_url"
-                    thumbnail-bg-color="expl.thumbnail_bg_color"
-                    num-views="expl.num_views"
-                    is-community-owned="expl.community_owned">
-                </exploration-summary-tile>
-              </div>
-              <div ng-if="userEditedExplorations.length === 0">
-                <em>This user hasn't edited any explorations yet.</em>
-              </div>
-            </md-content>
-          </md-tab>
-        </md-tabs>
+    <md-card class="oppia-profile-content-card">
+      <strong>Portfolio</strong>
+      <md-content class="oppia-profile-portfolio-container">
+          <exploration-summary-tile ng-repeat="expl in getExplorationsToDisplay()"
+            exploration-id="expl.id"
+            exploration-title="expl.title"
+            last-updated-msec="expl.last_updated_msec"
+            objective="expl.objective"
+            category="expl.category"
+            ratings="expl.ratings"
+            thumbnail-icon-url="expl.thumbnail_icon_url"
+            thumbnail-bg-color="expl.thumbnail_bg_color"
+            num-views="expl.num_views"
+            mobile-cutoff-px="610">
+          </exploration-summary-tile>
+
+        <div ng-if="userEditedExplorations.length === 0">
+          <em></br>This user hasn't created or edited any explorations yet.</em>
```

Comment:
Can it be replaced with a margin-top on the div containing it? In general, for spacing-related stuff, it's better to use CSS, especially since this isn't really semantically a line break.